app/src/main/kotlin/app/vitune/android/preferences/AppearancePreferences.kt

```diff
@@ -9,6 +9,7 @@ import app.vitune.core.ui.enums.ThumbnailRoundness
 object AppearancePreferences : GlobalPreferencesHolder() {
     var colorPaletteName by enum(ColorPaletteName.Dynamic)
     var colorPaletteMode by enum(ColorPaletteMode.System)
+    var usePureBlack by boolean(false)
```

Comment:
Not backwards compatible? Or am I missing something here?

Reply:
So I use a `private val` to "cover up" usePureBlack and revert to the old "multi-purpose-enum" approach from before?